systers_portal/common/tests/selenium/base.py

```diff
@@ -38,13 +39,14 @@ def setUp(self):
         country = Country.objects.create(name='Bar', continent='AS')
         self.location = City.objects.create(
             name='Foo', display_name='Foo', country=country)
-        with open('{0}{1}'.format(
-                  os.getcwd(),
+        with open('{0}{1}'.format(getcwd(),
                   '/systers_portal/common/tests/selenium/credentials.json')
                   ) as json_data:
             credentials = json.load(json_data)
-        self.user = User.objects.create(
-            username=credentials['username'], password=credentials['password'])
+        self.user = User()
+        self.user.username = credentials['username']
```

Comment:
Ok. Got, it, then in future we should log these errors as well. We are good for now.

Reply:
Instead of throwing the default python generated error, the sophisticated way to catch the python generated error here and throw it again, with more details of what went wrong, for easy debugging.